### PR TITLE
Ticket # 23 View All User Profiles https://trello.com/c/TXJDiNqo

### PR DESCRIPTION
**Pull request by Jordan-Twyman**
# Description

the user is an admin in the Tabloid application, when they select the User Profiles menu option they should be directed to the User Profiles list page and each user in the list should display the full name, the display name and the user type
And the list should be in ordered alphabetically by user display name.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x]  This change requires a documentation update

# Testing Instructions

`insert into UserProfile  (DisplayName, FirstName, LastName, Email, CreateDateTime, ImageLocation, UserTypeId)
VALUES ('aki', 'Aki', 'Endo', 'hi@here.net', '2022-03-01 11:15:45.730', NULL, 2);`

`insert into UserProfile  (DisplayName, FirstName, LastName, Email, CreateDateTime, ImageLocation, UserTypeId)
VALUES ('twy', 'J', 'Twy', 'j@email.come', '2022-03-01 11:15:45.730', NULL, 2);`

1.Please log in as the existing admin user admin@example.com
2. Navigate to USER PROFILES page
3. You should be able to view all existing users regardless of their User Type in alphabetical order based on "Display Name"
4. Log in as an Author user type using hi@here.net or j@email.come
5. Navigate to USER PROFILES page
6. The user should get a Not Found error message

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by cresuta** [comment]
Able to view list of user profiles while logged in as an Admin. Able to view the "Not Found" error message when logged in as hi@here.net

File: TabloidMVC/Views/Shared/_Layout.cshtml
Comment (jordan-castelloe) [inline TabloidMVC/Views/Shared/_Layout.cshtml]: I think it would be a better UI to not show this nav bar item at all if the user is logged in as a non-admin-- what do you think? Feels a little misleading to show the non-admins this link and then be like SIKE NOTHING HERE FOR YOU. Can you add a conditional here so that it only shows up if the user is an admin?

File: TabloidMVC/Views/Account/Index.cshtml
Comment (jordan-castelloe) [inline TabloidMVC/Views/Account/Index.cshtml:34]: Right now, the web page has the column headings as `FullName` and `DisplayName`. Let's add a space between those words just for the readability's sake :) You can add a `[Display(Name="")]` attribute on the model.

File: TabloidMVC/Repositories/UserProfileRepository.cs
Comment (jordan-castelloe) [inline TabloidMVC/Repositories/UserProfileRepository.cs]: Is this method used anywhere in this PR? If not, it probably shouldn't be on this PR

**Comment by heavenburdette1988** [comment]
Nicely done, everyone works correct.   The only change would be to move logout to the far right.

**Review by jordan-castelloe — CHANGES_REQUESTED** [review]
Hey! While everything technically worked as expected, there are a couple of UI changes I'd love to knock out before we merge this! Tag me on slack when these are done and let me know if you have questions.

A couple of general PR etiquette notes: 
1. I LOVE that you added a SQL script for seed data. When you include any code in your PR, it's helpful to put it in back ticks so that we can do a direct copy and paste. See [markdown styling guide](https://www.markdownguide.org/basic-syntax/) for options.
2. Be sure you include everything from the ticket in your testing instructions-- for example, that user profiles should appear in alphabetical order. It worked, but I had to go back to the ticket to see that I was supposed to test it :)

File: TabloidMVC/Views/Shared/_Layout.cshtml
Comment (Jordan-Twyman) [reply TabloidMVC/Views/Shared/_Layout.cshtml]: I made an attempt at this but had trouble accessing my repos in the Layout.cs.

File: TabloidMVC/Repositories/UserProfileRepository.cs
Comment (Jordan-Twyman) [reply TabloidMVC/Repositories/UserProfileRepository.cs]: I am using this in my Controllers Index conditional

**Review by jordan-castelloe — APPROVED** [review]
Looks great! You've got a tiny lil merge conflict but once you sort that out you're good to merge :)

**Pull request merged**
